README.markdown

```diff
@@ -56,4 +56,21 @@ The response returned by either of these requests Ruby (parsed JSON).
       }
 
 ## see also
-My other client library [comscore_ruby](https://github.com/msukmanowsky/comscore_ruby) for those of you looking to pull data from comscore as well.
\ No newline at end of file
+My other client library [comscore_ruby](https://github.com/msukmanowsky/comscore_ruby) for those of you looking to pull data from comscore as well.
+
+## contributing
```

Comment:
@rouanw are there specific instructions you wanted to add here? Otherwise think you can remove this.

Reply:
My intention was to create a `contributing` section with a `running the tests` sub-section. Users of the library probably won't be interested in how to run the tests, so I thought it would make sense to tuck it away in a `contributing` section.